internal/clients/http/image_builder/image_client.go

```diff
@@ -167,12 +175,24 @@ func (c *ibClient) checkCompose(ctx context.Context, composeID string) (*UploadS
 		return nil, fmt.Errorf("fetch image status call: %w", err)
 	}
 
-	if resp.JSON200.ImageStatus.Status != ImageStatusStatusSuccess {
+	return resp.JSON200, nil
+}
+
+func (c *ibClient) checkCompose(ctx context.Context, composeID string) (*UploadStatus, error) {
+	logger := logger(ctx)
+	logger.Trace().Msgf("Fetching image status %v from composes", composeID)
```

Comment:
```suggestion
	logger.Trace().Msgf("Fetching image status from compose %s", composeID)
```

is slightly better, not a blocker.